package.json

```diff
@@ -13,7 +13,7 @@
     "start": "npm run build && node dist/app.js",
     "start-dev": "nodemon --exec babel-node --presets env app.js",
     "pretest": "cross-env NODE_ENV=test  npm run db-migrate",
-    "test": "NODE_ENV=test mocha --require babel-register server/tests/**/*.test.js --exit",
+    "test": "NODE_ENV=test mocha --timeout 15000 --require babel-register server/tests/**/*.test.js --exit",
```

Comment:
Why is this `--timeout 15000` necessary?

Reply:
We need to extend the timeout of the test because Nodemailer takes a while to exchange client authentication before sending a mail and chai is set by default to exit with a timeout error after 2 seconds of waiting.

Reply:
Okay. 15000 is 15 seconds, right?